pipeline.py

```diff
@@ -103,23 +119,42 @@ def ilab_pipeline(
     final_eval_few_shots: int = 5,  # https://github.com/instructlab/instructlab/blob/v0.21.2/tests/testdata/default_config.yaml#L56
     final_eval_batch_size: str = "auto",  # https://github.com/instructlab/instructlab/blob/v0.21.2/tests/testdata/default_config.yaml#L52
     final_eval_merge_system_user_message: bool = False,  # https://github.com/instructlab/instructlab/blob/v0.21.2/src/instructlab/model/evaluate.py#L474
+    # General Evaluation Inputs
```

Comment:
i wonder if we should change the other eval params to conform to the prefix here, ie `eval_final_xyz`, `eval_mt_bench_xyz`, etc

Reply:
yeah I wasn't sure either, let's leave that for ux to figure out, they have a column for their suggestion in the api doc now